Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM busybox:1.24.2
+
+WORKDIR /app
+ENTRYPOINT /app/reaper.exe
+
+RUN addgroup -g 10001 app && \
+    adduser -G app -u 10001 -D -h /app -s /sbin/nologin app
+
+COPY version.json /app/version.json
+COPY reaper.exe /app/reaper.exe
```

Comment:
The .exe is named that way because there's already a [reaper folder in the repo](https://github.com/mozilla-services/reaper/tree/master/reaper).